packages/grafana-data/package.json

```diff
@@ -28,7 +28,7 @@
     "date-fns": "2.21.3",
     "eventemitter3": "4.0.7",
     "lodash": "4.17.21",
-    "marked": "4.0.0",
+    "marked": "4.0.5",
```

Comment:
need to bump the version of marked as they fixed a problem with their `main` resolution: see https://github.com/markedjs/marked/issues/2274